### ISSUE TITLE
[PNI] Privacy Policy links missing from page

### ISSUE THREAD

**Issue by benhohner** [mozilla/foundation.mozilla.org#7822]
On PNI product pages, the privacy policy links don't appear.
Links should appear, please confirm with design where.

**Comment by benhohner** [comment]
@kristinashu just to confirm, where should privacy policy links appear again? Bottom of page, or under privacy tab?

**Comment by kristinashu** [comment]
Yes, just below "User-friendly privacy information?", then there's the answer, then the links should be below.

<img width="771" alt="image" src="https://user-images.githubusercontent.com/1068783/141026572-8f53c15b-c648-4665-9c03-d2decd616d5a.png">
